### DISCUSSION TITLE
Changing default language

### DISCUSSION THREAD

**Discussion by jenzopr** [alfio-event/alf.io#1196] [Deployment]
Hi all! Did someone manage to change the default language (system-wide)? Any idea how to achieve that?

Best and thanks,
Jens

**Comment by cbellone** [comment]
Hi @jenzopr, and thanks for using alf.io! 

The current version doesn't have such setting. 

Default language is English everywhere and the Admin section is only available in English. This is expected to change in 2.0-M6, [see Roadmap](https://github.com/alfio-event/alf.io/issues/435) when we will implement the new Admin UI. Any help with that would be highly appreciated.

For the public section, you can specify a set of languages when [creating a new event](https://alf.io/docs/event-setup/create-in-person-event/#event-description). 
Users will then access content in their preferred language (automatically chosen according to their browser settings) if available.
